tests/playwright-test/test-modifiers.spec.ts

```diff
@@ -288,3 +288,20 @@ test('test.skip without a callback in describe block should skip hooks', async (
   expect(result.report.suites[0].suites[0].specs[0].tests[0].annotations).toEqual([{ type: 'skip', description: 'reason' }]);
   expect(result.output).not.toContain('%%');
 });
+
+
+test('test.skip should define a skipped test', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'a.test.ts': `
+      const { test } = pwt;
+      const logs = [];
+      test.skip('foo', () => {
```

Comment:
Please add a test where you call `test.skip(title, fn)` in the body of another test.